gemd/entity/value/inchi_value.py

```diff
@@ -22,14 +22,19 @@ def __init__(self, inchi=None):
 
     @property
     def inchi(self) -> str:
-        """Get the formula as a string."""
+        """Get the InChI as a string."""
         return self._inchi
 
     @inchi.setter
     def inchi(self, value: str):
+        """Set the InChI, correcting for some minor variations in format."""
         if value is None:
             self._inchi = None
         elif isinstance(value, str):
+            if not value.lower().startswith('inchi'):
+                value = f"InChI=1S/{value}"
```

Comment:
I've seen this with the `1s/` so I want to check for that too.  [Here's Sigma Aldrich for example](https://www.sigmaaldrich.com/US/en/product/aldrich/757136)


```suggestion
                if not value.lower().startswith("1s/") and not value.lower().startswith("1/"):
                    value = f"InChI=1S/{value}"
                else:
                    value = f"InChI={value}"
```

Reply:
Great catch; thanks!